packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -870,13 +875,20 @@ export const UnifiedDataTable = ({
   const schemaDetectors = useMemo(() => getSchemaDetectors(), []);
   const columnsVisibility = useMemo(
     () => ({
+      canDragAndDropColumns: defaultColumns ? false : canDragAndDropColumns,
```

Comment:
What is the role of `defaultColumns` here. Does this condition covers the scenario when `Summary` column is visible?

Reply:
@logeekal Yes, I noticed that columns are not movable right now when Summary is present. So I disabled the drag&drop too.